src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -2042,7 +2044,9 @@ case name:\
         ptr_outputs,
         ptr_pwl_segments_target);
 
-    connectInput(layer, ptr_inputs, num_data_bytes_in);
+    auto fused_to_layer = connectInput(layer, ptr_inputs, num_data_bytes_in);
+    // PWL will be fused with the previous layer and we need to use it's order id
+    layer->userValue.v_int = fused_to_layer.input->userValue.v_int;
```

Comment:
It makes sense, but is quite tricky, IMO we need better documentation 
with some example and focusing on different cases and orders including:
eg. conv -> pooling -> act
conv -> act -> pooliong
affine -> act